atomic_reactor/inner.py

```diff
@@ -562,6 +562,12 @@ def build_docker_image(self):
 
             signal.signal(signal.SIGTERM, signal.SIG_DFL)
 
+    def add_release_var(self, build_id):
```

Comment:
What's the thinking behind adding this to the DockerBuildWorkflow class, rather than having it as part of the bump_release plugin? I'm thinking this is something that only a single plugin would want to do.

Reply:
I initially wasn't sure where it was going to end up, so I put it in DockerBuildWorkflow so I could call it from whatever plugin would need it.

I'll go ahead and move it to pre_bump_release.